lib/pacemaker/pcmk_output.c

```diff
@@ -940,7 +940,7 @@ rsc_action_default(pcmk__output_t *out, va_list args)
             migrate_op = possible_matches->data;
         }
 
-        CRM_CHECK(next != NULL,);
+        CRM_CHECK(next != NULL,;);
```

Comment:
We actually have CRM_LOG_ASSERT() for this situation -- it's identical to CRM_CHECK() without any failure action. It could/should be used anywhere that looks like this